hyperspy/io_plugins/tvips.py

```diff
@@ -0,0 +1,347 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2021 The HyperSpy developers
+#
+# This file is part of  HyperSpy.
+#
+#  HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import re
+import logging
+from datetime import datetime
+
+import numpy as np
+import dask.array as da
+import dask
+
+from hyperspy.misc.array_tools import sarray2dict
+
+# Plugin characteristics
+# ----------------------
+format_name = 'tvips'
+description = ('Read support for TVIPS CMOS camera stream/movie files. Can be'
+               'used for in-situ movies or 4D-STEM datasets.')
+full_support = False
+# Recognised file extension
+file_extensions = ['blo', 'BLO']
```

Comment:
Should this be ```'tvips'``` and ```'TVIPS'```?

Reply:
yes, I haven't had time to look at this recently, I think at the beginning of october I can resume work on this